includes/fbproduct.php

```diff
@@ -448,15 +472,78 @@ public function get_fb_description() {
 		return apply_filters( 'facebook_for_woocommerce_fb_product_description', $description, $this->id );
 	}
 
+	/**
+	 * Get the rich text description for a product.
+	 *
+	 * This function retrieves the rich text product description based on the following logic:
+	 * 1. Check if the facebook rich text description is set and not empty.
+	 * 2. If the rich text description is available, use it as the preferred description.
+	 * 3. Otherwise, fall back to the plain text description made available by Woocommerce.
+	 *
+	 * @return string The rich text description for the product.
+	 */
+	public function get_rich_text_description() {
+		$rich_text_description = '';
+
+		// Check if the fb description is set as that takes preference
+		if ( $this->rich_text_description ) {
+			$rich_text_description = $this->rich_text_description;
+		} elseif ( $this->fb_description ) {
+			$rich_text_description = $this->fb_description;
+		}
+
+		// Try to get rich text description from post meta if description has been set
+		if ( empty( $rich_text_description ) ) {
+			$temp_rich_text_description = get_post_meta(
```

Comment:
Looks like `$temp_rich_text_description` is not necessary, you can just use `$rich_text_description`.